reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,76 @@
+---
+orphan: true
+---
+# 1.23.0
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.23.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
```

Comment:
@ajanon I think we should mention the switch to the Android 14 vendor image even if it is just to make people aware than that it serves much more information/value. From what I recall you have a todo to put the vendor image setup into the documentation too which then would nicely tie up.

Reply:
Yes, that makes sense.

Reply:
I have added a line. @ajanon let me know if there is anything else to be added or any links to reference